MimeTypeMap.cs

```diff
@@ -773,7 +807,7 @@ public static string GetExtension(string mimeType, bool throwErrorIfNotFound = t
                 throw new ArgumentException("Requested mime type is not registered: " + mimeType);
             }
 
-            return string.Empty;
+            return null;

```

Comment:
This would be a breaking change. I don't want to introduce any breaking changes. 
```suggestion
            return string.Empty;

```